src/App.tsx

```diff
@@ -21,78 +40,39 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={event => {
+              setItemsPerPage(+event.target.value);
+              handlePageChange(1);
+            }}
```

Comment:
Pls move the function inside `onChange` handler to a separate const and use as a callback here